layers/+distributions/spacemacs-base/local/centered-buffer-mode/centered-buffer-mode.el

```diff
@@ -0,0 +1,128 @@
+;;; centered-buffer-mode.el --- Minor mode for centering buffers.
+
+;; Copyright (C) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
```

Comment:
The idea was to attribute the package to you so you'll have to make it into a quality code. But you noticed it too early. Now I have to change the attribution. :imp: